modules/controller/lib/verify_content_type.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(request) {
+  var contentType = 'application/vnd.api+json';
+  return (
+    request.header('content-type') &&
```

Comment:
If you use `request.headers['content-type']` this will work for both hapi and express.